Kinvey/Kinvey/FindOperation.swift

```diff
@@ -164,7 +164,13 @@ internal class FindOperation<T: Persistable>: ReadOperation<T, [T], Swift.Error>
                         self.executeLocal(completionHandler)
                     }
                 } else {
-                    let entities = [T](JSONArray: jsonArray)
+                    func convert(_ jsonArray: [JsonDictionary]) -> [T] {
+                        let startTime = CFAbsoluteTimeGetCurrent()
+                        let entities = [T](JSONArray: jsonArray)
+                        log.debug("Time elapsed: \(CFAbsoluteTimeGetCurrent() - startTime) s")
```

Comment:
@heyzooi how do we know what method this "time elapsed" comes from? Is there any info in the debug logs indicating that this is from JSON conversion in the `FindOperation`?